__tests__/app.test.js

```diff
@@ -115,4 +115,52 @@ describe('/api/reviews/:review_id', () => {
                 });
         });
     });
+});
+
+describe('/api/reviews/:review_id/comments', () => {
+    describe("GET", () => {
+        test("should return the review from the table if the Id is valid", () => {
+            return request(app)
+                .get("/api/reviews/2/comments")
+                .expect(200)
+                .then(({ body }) => {
+
+                    //expect(body.review.length).toBeGreaterThan(0)
+                    expect(body.review[0]).toMatchObject({
+                        comment_id: 1,
+                        body: 'I loved this game too!',
+                        review_id: 2,
+                        author: 'bainesface',
+                        votes: 16,
+                        created_at: '2017-11-22T12:43:33.389Z'
+                    })
+
+                })
+        });
+        test("should return an empty array if there are no comments for the given review", () => {
+            return request(app)
+                .get("/api/reviews/1/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.review.length).toEqual(0)
+                    expect(body.review).toEqual([])
+                })
+        });
+        test("should return a 404 status if the Id is invalid", () => {
+            return request(app)
+                .get("/api/reviews/10000/comments")
+                .expect(404)
+                .then(({ body }) => {
+                    expect(body.msg).toEqual("review does not exist");
+                });
+        });
+        test("should return a 404 status if the Id is invalid", () => {
+            return request(app)
+                .get("/api/reviews/a/comments")
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body.msg).toEqual("Bad request");
+                });
+        });
```

Comment:
Solid testing for errors - make sure you're using `.toBe` for primitive data types